rust/src/model/message.rs

```diff
@@ -267,46 +292,57 @@ impl MessageView {
         }
     }
 
+    /// get message id
     pub fn message_id(&self) -> &str {
         &self.message_id
     }
 
+    /// get topic of message
     pub fn topic(&self) -> &str {
         &self.topic
     }
 
+    /// get message body
     pub fn body(&self) -> &[u8] {
         &self.body
     }
 
+    /// get message tag
     pub fn tag(&self) -> Option<&str> {
         self.tag.as_deref()
     }
 
+    /// get message keys
     pub fn keys(&self) -> &[String] {
         &self.keys
     }
 
+    /// get message properties
     pub fn properties(&self) -> &HashMap<String, String> {
         &self.properties
     }
 
+    /// get message group of fifo message
     pub fn message_group(&self) -> Option<&str> {
         self.message_group.as_deref()
     }
 
+    /// get delivery timestamp of delay message
     pub fn delivery_timestamp(&self) -> Option<i64> {
```

Comment:
Return Option<Instant>?

Reply:
Instant seems could not be created by ourselves.

> A measurement of a monotonically nondecreasing clock. Opaque and useful only with [Duration](https://doc.rust-lang.org/std/time/struct.Duration.html).

Reply:
OK